data/onPostBuild/generateMarkdownFooter.test.ts

```
@@ -0,0 +1,450 @@
+      expect(footer).toContain('## Page Navigation');
+      expect(footer).not.toContain('## Related Topics');
+      expect(footer).toContain('## Documentation Index');
+    });
+
+    it('should always include Documentation Index section', () => {
+      const navContext: NavContext = { siblings: [] };
+      const footer = generateNavigationFooter(navContext, siteUrl);
+
+      expect(footer).toContain('## Documentation Index');
+      expect(footer).toContain('To discover additional Ably documentation:');
+      expect(footer).toContain('1. Fetch [llms.txt](https://ably.com/llms.txt) for the canonical list of available pages.');
```

ESLint

[error] Replace `'1.·Fetch·[llms.txt](https://ably.com/llms.txt)·for·the·canonical·list·of·available·pages.'` with `⏎········'1.·Fetch·[llms.txt](https://ably.com/llms.txt)·for·the·canonical·list·of·available·pages.',⏎······`

(prettier/prettier)